cylc/flow/task_events_mgr.py

```diff
@@ -1325,9 +1325,9 @@ def _process_message_failed(self, itask, event_time, message, forced):
         else:
             # There is an execution retry lined up.
             timer = itask.try_timers[TimerFlags.EXECUTION_RETRY]
-            self._retry_task(itask, timer.timeout)
             delay_msg = f"retrying in {timer.delay_timeout_as_str()}"
-            LOG.warning(f"[{itask}] {delay_msg}")
+            LOG.warning(f'[{itask}] => {TASK_OUTPUT_FAILED} ({delay_msg})')
```

Comment:
LGTM.

Could we get the actual failure message into this as it would appear if retry delays were not being used.

(e.g. if you hit wallclock limit you might expect to see XCPU here)

Reply:
> Needs tests.

Yes, wanted you to ok the approach